src/sync/installation.ts

```diff
@@ -144,9 +144,7 @@ const updateJobStatus = async (
 			}
 		});
 
-		const delay = Number(process.env.LIMITER_PER_INSTALLATION) || 1000;
-
-		queues.installation.add(job.data, { delay });
+		queues.installation.add(job.data);
```

Comment:
no idea why there was a 1 second delay for each installation job, with proper rate limiting handling, this shouldn't matter